packages/web3-wrapper/README.md

```diff
@@ -1,10 +1,50 @@
-Web3 wrapper
+@0xproject/web3-wrapper
 ------
 
 Wrapped version of web3 with nicer interface to be used across 0x projects and packages
 
-## Install
+## Installation
 
 ```bash
 yarn add @0xproject/web3-wrapper
 ```
+
+## Usage
+
+TODO - Custom documentation page is in progress.
```

Comment:
Ditto here. Just show an example of importing it, instantiating it and calling a single method on it.